fxdc/fields.py

```
@@ -0,0 +1,81 @@
+    @property
+    def blank(self) -> bool:
+        return self.__blank
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self.name = name
+
+    def __get__(self, instance: object, owner: object) -> T:
+        value: T = instance.__dict__.get(self.name, None)
+        return value
+
+    def __set__(self, instance: object, value: Optional[T]) -> None:
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)